### PR TITLE
feat: autotag geoip traits

### PR DESCRIPTION
**Pull request by Rileran**
## Description

Add autotag for traits in maxmind geoip (`is_anonymous_proxy` & `is_satellite_provider`).

Those traits are only available in the GeoIP2 commercial database, it is therefore impossible to write test for theses tags.

### CODE REVIEW

File: curiefense/curieproxy/rust/curiefense/src/tagging.rs [hunk 1]
```diff
@@ -232,6 +232,18 @@ pub fn tag_request(
             tags.insert_qualified("geo-asn", &sasn, Location::Ip);
         }
     }
+
+    if let Some(is_anonymous_proxy) = rinfo.rinfo.geoip.is_anonymous_proxy {
+        if is_anonymous_proxy {
+            tags.insert("maxming-anonymous-proxy", Location::Ip)
```
Comment (bartavelle) [inline curiefense/curieproxy/rust/curiefense/src/tagging.rs:238]: typo :)

File: curiefense/curieproxy/rust/curiefense/src/tagging.rs [hunk 2]
```diff
@@ -232,6 +232,18 @@ pub fn tag_request(
...
+        }
+    }
+    if let Some(is_satellite_provider) = rinfo.rinfo.geoip.is_satellite_provider {
+        if is_satellite_provider {
+            tags.insert("maxming-satellite-provider", Location::Ip)
```
Comment (bartavelle) [inline curiefense/curieproxy/rust/curiefense/src/tagging.rs:243]: typo :)

**Review by bartavelle — CHANGES_REQUESTED** [review]

File: curiefense/curieproxy/rust/curiefense/src/tagging.rs (hunk 1 above)
Comment (bartavelle) [reply curiefense/curieproxy/rust/curiefense/src/tagging.rs:238]: change to `mm-anon`

File: curiefense/curieproxy/rust/curiefense/src/tagging.rs (hunk 2 above)
Comment (bartavelle) [reply curiefense/curieproxy/rust/curiefense/src/tagging.rs:243]: change to `mm-sat`

File: curiefense/curieproxy/rust/curiefense/src/utils/mod.rs [hunk 3]
```diff
@@ -259,6 +261,8 @@ impl GeoIp {
         if let Some(network) = self.network {
             out.insert("network", json!(format!("{}", network)));
         }
+        out.insert("is_anonymous_proxy", json!(self.is_anonymous_proxy));
```
Comment (bartavelle) [inline curiefense/curieproxy/rust/curiefense/src/utils/mod.rs:264]: `is_anon`

File: curiefense/curieproxy/rust/curiefense/src/utils/mod.rs [hunk 4]
```diff
@@ -259,6 +261,8 @@ impl GeoIp {
...
+        out.insert("is_satellite_provider", json!(self.is_satellite_provider));
```
Comment (bartavelle) [inline curiefense/curieproxy/rust/curiefense/src/utils/mod.rs:265]: `is_sat`

File: curiefense/curieproxy/rust/curiefense/src/tagging.rs (hunk 1 above)
Comment (Rileran) [reply curiefense/curieproxy/rust/curiefense/src/tagging.rs]: fixed!

File: curiefense/curieproxy/rust/curiefense/src/utils/mod.rs (hunk 3 above)
Comment (Rileran) [reply curiefense/curieproxy/rust/curiefense/src/utils/mod.rs]: fixed!

File: curiefense/curieproxy/rust/curiefense/src/tagging.rs (hunk 2 above)
Comment (Rileran) [reply curiefense/curieproxy/rust/curiefense/src/tagging.rs]: fixed!

File: curiefense/curieproxy/rust/curiefense/src/utils/mod.rs (hunk 4 above)
Comment (Rileran) [reply curiefense/curieproxy/rust/curiefense/src/utils/mod.rs]: fixed!

**Pull request closed without merging**